packages/framework/src/Model/Order/Processing/OrderProcessorMiddleware/ApplyNominalPromoCodeMiddleware.php

```diff
@@ -62,10 +62,13 @@ public function createAndAddOrderItemData(
         PromoCodeLimit $promoCodeLimit,
         OrderProcessingData $orderProcessingData,
     ): void {
+        $totalApplicableProductsPrice = $this->calculateTotalApplicableProductsPriceAmountWithVat($orderData, $validProductIds);
```

Comment:
```suggestion
        $totalApplicableProductsPriceAmountWithVat = $this->calculateTotalApplicableProductsPriceAmountWithVat($orderData, $validProductIds);
```

same in the `createDiscountOrderItemData()` method definition 😇

Reply:
Ah, the hard part of programmers life - naming variables correctly. I hope it is ok now. 😁